odoo80/Dockerfile

```diff
@@ -25,12 +23,10 @@ RUN apt-get update -q && apt-get upgrade -q \
     postgresql-common \
     libpq-dev \
     libldap2-dev \
-    libsasl2-dev \
     libxml2-dev \
     libxslt1-dev  \
     xsltproc \
     xmlstarlet \
-    openssl \
     python-libxml2 \
     phantomjs \
     bash-completion
```

Comment:
@ruiztulio 
I got it thank you.

I saw the follow trends:
[nginx](https://github.com/nginxinc/docker-nginx/blob/aecb18605af6a33c1968c51df44e01e5f5a821e6/Dockerfile#L11)
[jenkinsci](https://github.com/jenkinsci/docker/blob/62de121e41ddd92ef34e9999162932334b05f0d7/Dockerfile#L3)
[apache](https://github.com/docker-library/httpd/blob/1f1f7d39d5fe5aebeedea6872786b4e3ce0ebcc9/2.4/Dockerfile#L22)

All them have a `rm` in the same step of `apt-get update`.
My theory is that they want avoid have a layer with this files versioning and other layer with this files deleted.
I think that is like as `git add file` and `git rm file` making a entry in `.git` database.
What do you think?